metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/configs/HiveConnectorClientConfig.java

```diff
@@ -152,4 +175,13 @@ private HiveConf getDefaultConf() {
         result.setBoolean(HiveConfigConstants.HIVE_STATS_AUTOGATHER, false);
         return result;
     }
+
+    private int getDataStoreTimeout(final ConnectorContext connectorContext) {
+        int result = DEFAULT_DATASTORE_READ_TIMEOUT;
+        try {
+            result = Integer.parseInt(
+                connectorContext.getConfiguration().get(HiveConfigConstants.JAVAX_JDO_DATASTORETIMEOUT));
+        } catch (final Exception ignored) { }
+        return result;
```

Comment:
What if the passing value is 0 or a very big number? Should we have error check?

Reply:
Since this is one-time during initialization and also because it is getting defaulted to 60 secs, I ignore it.